frontend/src/component/layout/MainLayout/NavigationSidebar/NavigationSidebar.tsx

```diff
@@ -83,25 +87,65 @@ const EnterprisePlanBadge = () => (
     </Tooltip>
 );
 
-const FullListItem: FC<{ href: string; text: string; badge?: ReactNode }> = ({
+const FullListItem: FC<{
```

Comment:
Extract reusable component. FullListItem has icon + text + optional badge as opposed to MiniListItem which only has icons with tooltip